src/lib/scene-internal.ts

```diff
@@ -744,7 +748,6 @@ export class SceneInternal implements Renderer {
    * or offset is changed, this method is invoked.
    */
   private handleViewChange() {
-    this.isViewInitialized = true;
```

Comment:
This was causing `InternalError`s to be thrown when the scene's `scale` and/or `offset` were being altered without actual initialization occurring.